sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPDiskEncryptionSetTests.cs

```diff
@@ -35,13 +35,13 @@ public void DiskEncryptionSet_List()
         [Fact]
         public void DiskEncryptionSet_CreateDisk()
         {
-            DiskEncryptionSet_CreateDisk_Execute("DiskEncryptionSet_CreateDisk", location: supportedZoneLocation);
+            DiskEncryptionSet_CreateDisk_Execute("DiskEncryptionSet_CreateDisk", location: "eastus2");
```

Comment:
We shoudl avoid hard-coding locations in the tests, except as defaults.  There are helpers to look up valid locations and default to a particular location if it is in the list.  This allows tests to run in multiple clouds and cuts down on code modifications when runnign the tests.  This is not a blocker, just an FYI on maintaining these tests

Reply:
@markcowl , I agree, this is just what we were able to figure out on our own. The DiskRP team owns this and will be called upon to improve these tests.